.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
Let's fix it to `20.11.0` for avoiding constant updates of minimal versions in our local environment 😄

Reply:
I'm not sure about this, because if we add minor and patch versions then we'll have tons of different node versions in our computers depending on the specific version of every repo. What's the benefit of adding the minor and patch version and not use only the major version?

Reply:
I'd agree with @inigomarquinez here. I think the best to define the minimum required LTS version.